pkg/arm/data/master-startup.sh

```diff
@@ -572,6 +562,34 @@ spec:
     name: master-cloud-provider
 EOF
 
+{{ if not (eq "true" .Config.RunSyncLocal) }}
+if [[ "$(hostname)" == "master-000000" ]]; then
```

Comment:
I really dont like this...  Partially related to this: https://github.com/openshift/openshift-azure/issues/175
Any issues with master0 takes out our syn process down.

Reply:
100% agree.  It is abominably hard to get the sync pod to run at this stage in the cluster startup by any other means.  This is a stopgap measure.  If we ran the SDN workload as static pods, then nodes would immediately come up ready, which would speed setup time, and would then enable us to run the sync pod as a deployment.  #242.